src/core/adapters/personalization.ts

```diff
@@ -0,0 +1,117 @@
+import Actions from '../actions';
+import Configuration from '../configuration';
+import Selectors from '../selectors';
+import Store from '../store';
+import ConfigAdapter from './configuration';
+
+namespace Personalization {
```

Comment:
In general in this file we have a bunch of references to `variant` which is a very specific and different concept from what we want to be referring to here which is `refinements`

everywhere that we have `key` and `variant` I would prefer the wording `field` and `value` which is what we use everywhere else

Reply:
Fixed.